### PR TITLE
Added beima prefix to FT

### LINKED ISSUE #542: Switch backend FT to use beima prefix
References user story #434

We need to switch over the backend FT to use the beima prefix instead of the api prefix.

### PR DESCRIPTION
**Pull request by josephmoore99**
Closes #542

Adds the beima url prefix to the backend FT

### CODE REVIEW

File: BEIMA.Backend.FT/BeimaClient.cs [hunk 1]
```diff
@@ -194,7 +194,7 @@ public async Task<HttpResponseMessage> SendMultiPartRequest(string route, Object
         /// <returns>The device with the given id.</returns>
         public async Task<Device> GetDevice(string id)
         {
-            var response = await SendRequest($"api/device/{id}", HttpVerb.GET);
+            var response = await SendRequest($"device/{id}", HttpVerb.GET);
             var content = await response.Content.ReadAsStringAsync();
```
Comment (AshlynAdamson) [inline BEIMA.Backend.FT/BeimaClient.cs:198]: I'm not sure the api part was removed, but just the beima part was added

Comment (josephmoore99) [reply BEIMA.Backend.FT/BeimaClient.cs:198]: When I run it on my machine it seems to be replaced? 
image

Comment (AshlynAdamson) [reply BEIMA.Backend.FT/BeimaClient.cs:198]: Ah okay, good to know

**Review by Kenny-Miller — APPROVED** [review]

File: BEIMA.Backend.FT/BeimaClient.cs (hunk 1 above)
Comment (josephmoore99) [reply BEIMA.Backend.FT/BeimaClient.cs:198]: Looks like I just needed to add in an extra slash was the problem

**Pull request merged**